gamemodes/terrortown/gamemode/shared/sh_door.lua

```diff
@@ -0,0 +1,184 @@
+--- DOORS MODULE STUFF ---
+
+---
+-- @module door
+-- @author Mineotopia
+-- @desc A bunch of functions that handle all doors found on a map
+
+door = {}
+door.__doors = {}
```

Comment:
We never did anything related to this var naming, uff. Welcome to C

Reply:
How do I name an internal car to make it clear that it should not be accessed directly. I guess I could make an internal local variable

Reply:
That should be the way to prevent access